ivy/functional/backends/numpy/experimental/statistical.py

```diff
@@ -139,12 +139,7 @@ def median(
         keepdims=keepdims,
         out=out,
     )
-    if input.dtype in [np.uint64, np.int64, np.float64]:
```

Comment:
was this type-casting unnecessary? ig `numpy.median` was producing output of an incorrect dtype which is why this type casting was added? Not completely sure though

Reply:
> was this type-casting unnecessary? ig `numpy.median` was producing output of an incorrect dtype which is why this type casting was added? Not completely sure though

when we can say that the return type is incorrect ?

Reply:
It varies from function to function, but I think for `median`, the output `dtype` should intuitively be the same as the input `dtype`?